openapi/components/schemas/Storefront/Invoice.yaml

```
@@ -0,0 +1,159 @@
+type: object
+required:
+  - currency
+properties:
+  id:
+    description: The invoice identifier string
+    readOnly: true
+    allOf:
+      - $ref: ../ResourceId.yaml
+  websiteId:
+    description: The website identifier string
+    readOnly: true
+    allOf:
+      - $ref: ../ResourceId.yaml
+  invoiceNumber:
+    description: An auto-incrementing number based on the sequence of invoices for any particular customer.
+    readOnly: true
+    type: integer
+  subscriptionId:
+    description: The related Subscription's ID if available, otherwise null
+    readOnly: true
+    allOf:
+      - $ref: ../ResourceId.yaml
+  currency:
+    allOf:
+      - $ref: ../CurrencyCode.yaml
+  amount:
+    description: The Invoice's amount
+    type: number
+    format: double
+    readOnly: true
+  amountDue:
+    description: The Invoice's due amount
+    type: number
+    format: double
+    readOnly: true
+  subtotalAmount:
+    description: The Invoice's subtotal amount
+    type: number
+    format: double
+    readOnly: true
+  discountAmount:
+    description: The Invoice's discounts amount
+    type: number
+    format: double
+    readOnly: true
+  taxAmount:
+    description: The Invoice's taxes amount
+    type: number
+    format: double
+    readOnly: true
+  shippingAmount:
+    description: The Invoice's shipping amount
```

Invoice is not a proper name. The problem exists everywhere in this PR.
```suggestion
    description: The invoice's shipping amount
```